examples/Microsoft.Cache_redisEnterprise@2025-04-01/main.tf

```diff
@@ -27,17 +27,23 @@ resource "azapi_resource" "resourceGroup" {
 }
 
 resource "azapi_resource" "redisEnterprise" {
-  type      = "Microsoft.Cache/redisEnterprise@2022-01-01"
+  type      = "Microsoft.Cache/redisEnterprise@2025-04-01"
   parent_id = azapi_resource.resourceGroup.id
   name      = var.resource_name
   location  = var.location
   body = {
     properties = {
       minimumTlsVersion = "1.2"
+      encryption        = {}
+      highAvailability  = "Enabled"
     }
     sku = {
-      capacity = 2
-      name     = "Enterprise_E100"
+      name = "Balanced_B0"
+    }
+
+    identity = {
```

Comment:
Please move the identity block to the root level.

Refs: https://registry.terraform.io/providers/azure/azapi/latest/docs/resources/resource#identity-1